internal/blocksync/reactor.go

```diff
@@ -76,7 +76,7 @@ type Reactor struct {
 	stateStore sm.Store
 
 	blockExec   *sm.BlockExecutor
-	store       *store.BlockStore
+	store       sm.BlockStore
```

Comment:
This is just for my understanding, why is the internal/store replaced with the store from internal/state?

Reply:
When I was navigating this code, I saw other places were using the blockstore interface, rather than the object directly... so I thought I'd change this for consistency... but I might be missing the bigger picture